resource/StandardResources.md

```diff
@@ -0,0 +1,94 @@
+# Standard Resources
+
+This page lists the standard resource types in OpenCensus. For more details on how resources can 
+be combined see [this](Resource.md).
+
+OpenCensus defines these fields.
+ * [Compute Unit](#compute-unit)
+   * [Container](#container)
+ * [Deployment Service](#deployment-service)
+   * [Kubernetes](#kubernetes)
+ * [Compute Instance](#compute-instance)
+   * [Host](#host)
+ * [Environment](#environment)
+   * [Cloud](#cloud)
+   * [Cluster](#cluster)
+
+## TODOs
+* Add logical compute units: Service, Task - instance running in a service.
+* Add more compute units: Process, Lambda Function, AppEngine unit, etc.
+* Decide if lower case strings only.
+* Consider to add optional/required for each label.
+
+## Compute Unit
+Resources defining a compute unit (e.g. Container, Process, Lambda Function).
+
+### Container
+**type:** `container`
+
+**Description:** A container instance. This resource can be [merged](Resource.md#Merging) with a
+deployment service resource, a compute instance resource, and an environment resource.
+
+| Label  | Description  | Example  |
+|---|---|---|
+| container.name | Container name. | `opencenus-autoconf` |
+| container.image.name | Name of the image the container was built on. | `gcr.io/opencensus/operator` |
+| container.image.tag | Container image tag. | `0.1` |
+
+## Deployment Service
+Resources defining a deployment service (e.g. Kubernetes).
+
+### Kubernetes
+**type:** `k8s`
+
+**Description:** A Kubernetes pod resource. This resource can be [merged](Resource.md#Merging) with
+a compute instance resource, and/or an environment resource.
+
+| Label  | Description  | Example  |
+|---|---|---|
+| k8s.cluster.name | The name of the cluster that the container is running in. | `opencensus-cluster` |
+| k8s.namespace.name | The name of the namespace that the container is running in. | `default` |
+| k8s.pod.name | The name of the pod that the container is running in. | `opencensus-pod-autoconf` |
+
+## Compute Instance
+Resources defining a computing instance (e.g. host).
+
+### Host
```

Comment:
is an app runs in VM without any containerization - will the Host type describe this VM properties? Or it will be a VM compute unit?

Reply:
Same goes for mobile apps. Is mobile OS will be described as Host or with the new type `Device`?

Reply:
For first comment: The host will describe the VM instance. Inside the VM if you run directly a "process" we will define a "process" compute unit, see TODO.

For second comment: I think the new type Device should describe that.